apps/s_server.c

```diff
@@ -1387,10 +1387,9 @@ int s_server_main(int argc, char *argv[])
         case OPT_STATUS_URL:
 #ifndef OPENSSL_NO_OCSP
             s_tlsextstatus = 1;
-            if (!OCSP_parse_url(opt_arg(),
-                                &tlscstatp.host,
-                                &tlscstatp.port,
-                                &tlscstatp.path, &tlscstatp.use_ssl)) {
+            if (!!OSSL_HTTP_parse_url(opt_arg(),
```

Comment:
I don't think you meant `!!` here

Reply:
Oops, of course - this was a pretty bad copy&paste mistake. Fixed.